dbt_platform_helper/templates/pipelines/codebase/overrides/buildspec.deploy.yml

```diff
@@ -8,8 +7,28 @@ env:
   variables:
     COLOR: false
     CI: true
-
 phases:
   build:
     commands:
-      - /work/cli deploy --send-notifications
+      - |
+        environments="dev,staging,prod"
```

Comment:
is this always going to be one of these 3 envs? worth parametrizing or not? just in case somebody spins up `live` rather than prod or one env is missing?

Reply:
True, will look into it 👍🏻